### PR TITLE
Run single node live CD flow on installer presubmit

### PR DESCRIPTION
**Pull request by omertuc**
Based on #14552

### CODE REVIEW

File: ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml [hunk 1]
```diff
@@ -2008,6 +2008,75 @@ presubmits:
...
+    - master
+    cluster: build02
+    context: ci/prow/e2e-metal-single-node-live-iso
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-installer-master-e2e-metal-single-node-live-iso
```
Comment (jstuever) [inline ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml:2023]: Add `optional: true`

File: ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml [hunk 2]
```diff
@@ -2008,6 +2008,75 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-ipi-virtualmedia,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (jstuever) [inline ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml:2012]: `always_run: false`

**Review by jstuever — CHANGES_REQUESTED** [review]
I don't think this should be blocking for all PRs.

**Comment by omertuc** [comment]
> I don't think this should be blocking for all PRs.

Right, of-course. Just forgot to `git add`. Fixing.

**Comment by jstuever** [comment]
/assign @staebler
I don't know if we need more rehearsals to pass, but we probably wont' get them all in one go.

**Pull request merged**